src/test/java/com/google/api/codegen/testdata/discoveries/mvvm_java/java_adexchangebuyer.v1.4.json.baseline

```diff
@@ -1650,6 +1652,7 @@ import java.security.GeneralSecurityException;
 public class AdExchangeBuyerExample {
   public void run() throws IOException, GeneralSecurityException {
     // TODO: Assign values to the desired fields of `requestBody`.
+    // Only the assigned fields will be changed.
```

Comment:
I'm not sure, but I figure if the patch verb is used then the request should follow patch semantics, and we should be right regardless.
The alternative is that the method is `.patch()` but the verb is not patch... which sounds like an API design error.